tests/desktop_dinosaurs.pm

```diff
@@ -0,0 +1,67 @@
+use base 'installedtest';
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+    my $sdk_nightly_repo = 'http://endlessm.github.io/eos-knowledge-lib/eos-sdk-nightly.flatpakrepo';
+
+    check_desktop_clean();
+    type_very_safely("gnome-terminal\n");
+    assert_screen('desktop_terminal', 5);
+
+    type_string("flatpak remote-add --from eos-sdk-nightly " . $sdk_nightly_repo . "\n");
```

Comment:
I see these lines duplicated in the three added tests. They are the same with the exception of the app ID. So this code should be generalized in one or more functions, with app ID as parameter. https://github.com/endlessm/eos-openqa-tests/blob/dd770a9490ce4b3fafcd5054ca379ddd6f700547/tests/desktop_dinosaurs.pm#L14-L28